src/hooks/loading.ts

```diff
@@ -61,5 +62,11 @@ export const useLoading = <T>(
     }
   }
 
-  return { data, isLoading, isLoadingError, isEmpty, reload, update }
+  const reset = () => {
+    setIsLoading(false)
+    setIsLoadingError(false)
+    setData(initialState as T)
+  }
```

Comment:
what if loadOnMount = true? which behaviour expects here?

Reply:
should be fine